### ISSUE TITLE
Command line interface

### ISSUE THREAD

**Issue by bfsujason** [memphis-iis/cohmetrix-issues#54]
The README file indicates that cohmetrix can be excecuted using command line:

CohMetrixCoreCLI.exe <input file or directory> <output CSV>

I installed both cohmetrixcore-1.0.2-windows-installer.exe and cohmetrixcorecli-1.00-windows-installer.exe, but can't find [CohMetrixCoreCLI.exe].

Where is it located and how to run the command?

Thanks.

**Comment by JRustyHaner** [comment]
@bfsujason the installer puts the exe file inside the programs file (x86) folder. You can look for it there.

**Comment by bfsujason** [comment]
Thank you for your reply.

I checked the installation directory: C:\Program Files (x86)\CohMetrixCore\bin, but can't find CohMetrixCoreCLI.exe there.
There is only one .exe file in the directory: CohMetrixCore Desktop.exe

I used the latest installser: cohmetrixcore-1.0.2-windows-installer.exe.

![Screenshot 2024-02-12 104727](https://github.com/memphis-iis/cohmetrix-issues/assets/6072421/ee171963-3a47-4a72-b338-eee8a61c5b81)
